.gitignore

```diff
@@ -1,5 +1,3 @@
-/*
```

Comment:
Ignore all files by default is inconvenient.

Reply:
In the early attempts, we try using the `blacklist mode`, but unexpected generated files (configs, cache, test files, etc) appears frequently and bother our development, so it is the whitelist mode now.

Reply:
Files generated by well-known tools and frameworks often follow established conventions, which can be found in resources like [this GitHub repository](https://github.com/github/gitignore/blob/main/Qt.gitignore).

Similarly, files we generate should also follow our own conventions to maintain consistency and organization. For example, we could always store built installers in the `dist` directory and keep test caches within `cache` and its subdirectories.

Reply:
I strongly recommend using the blacklist mode of .gitignore. However, if this change disrupts established conventions or workflows for others, I’m happy to remove the commit for .gitignore and maintain it in my fork instead.